### PR TITLE
openjade: fix broken code

### PR DESCRIPTION
**Pull request by alarcher**
*Blurb*

### CODE REVIEW

File: components/developer/openjade/manifests/sample-manifest.p5m [hunk 1]
```diff
@@ -10,7 +10,7 @@
 #
 
 #
-# Copyright 2017 <contributor>
+# Copyright 2018 <contributor>
```
Comment (AndWac) [inline components/developer/openjade/manifests/sample-manifest.p5m]: Haha, when did you do this?

**Review by AndWac — APPROVED** [review]

File: components/developer/openjade/patches/openjade-05-static-iid.patch [hunk 2]
```diff
@@ -0,0 +1,73 @@
+Symbols of static variables are deleted from object files and anyway can be replaced by plain static inline calls
```
Comment (pyhalov) [inline components/developer/openjade/patches/openjade-05-static-iid.patch:1]: Is it a known issue?

Comment (alarcher) [reply components/developer/openjade/patches/openjade-05-static-iid.patch:1]: I have not seen anyting like that but other distros maintain their own fork...
Probably a peculiarity of our linker, at least for the virtual function it is clear.
In any case the use of static members is ugly.

File: components/developer/openjade/manifests/sample-manifest.p5m (hunk 1 above)
Comment (alarcher) [reply components/developer/openjade/manifests/sample-manifest.p5m]: I travelled back in time :)

File: components/developer/openjade/patches/openjade-05-static-iid.patch (hunk 2 above)
Comment (alarcher) [reply components/developer/openjade/patches/openjade-05-static-iid.patch:1]: If you are happy with it, feel free to merge.

**Pull request merged**